tests/ContainerTest.php

```diff
@@ -14,7 +14,7 @@ public function test_container_build()
 
         $volume = K8s::volume()->awsEbs('vol-1234', 'ext3');
 
-        $container->setImage('nginx', '1.4')
+        $container->setImage('public.ecr.aws/docker/library/nginx', '1.23')
```

Comment:
I always prefer a qualified repository  (to avoid the ambiguous defaulting and moreso the ubiquitious default of docker.io with a rate limit), but I would have liked to have a qualified default and a way to send it in as a parameter to phpunit (or a search order to the minikube), but I didn't look for that.

All images are prevalent, popular, publiic ones, so it was easy to set to public.ecr.aws to avoid the rate limits in local testing (I don't know if it's an issue in a GH action or not).